api_tests/lib/ledger_runner.ts

```diff
@@ -45,6 +45,7 @@ export class LedgerRunner {
                         await getPort({ port: 18444 }),
                         await getPort({ port: 18443 })
                     );
+                    global.bitcoind = instance;
```

Comment:
Why can we not use existing way of accessing data from the bitcoind instance on the global object?

Reply:
Can you extrapolate please, I don't see how to?

Reply:
@thomaseizinger I'm going to leave this for a follow up PR, this is not trivial to resolve unless I'm missing something.

Reply:
I'll tackle this once #2015 is done.

Reply:
What I meant was: we already expose the bitcoind instance through global.ledgers.bitcoin